### PR TITLE
PAAS-1535: fix jexperience broken when karaf password contains consec…

### PR DESCRIPTION
**Pull request by adoux**
…utive dollar signs

JIRA issue: https://jira.jahia.org/browse/PAAS-1535

Short description:
I tested with `Pa$$$w$$o$rd1'2"3\` and it works so I really hope it's going to be the last PR :pray:

### CODE REVIEW

File: packages/jcustomer/set-unomi-root-password.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ onInstall:
       nodeId: ${@i.id}
 
   - updateJahia:
-      new_password: ${settings.rootpwd}
+      new_password: ${settings.rootpwd.toBase64()}
```
[the remark this answers is not in the document]
Comment (adoux) [reply packages/jcustomer/set-unomi-root-password.yml]: Done!

File: mixins/jahia.yml [hunk 2]
```diff
@@ -788,7 +788,7 @@ actions:
   checkJexperienceCfg:
     # Parameters:
     #   - jcustomerDns: jahia env name to fetch info from
-    #   - jcustomerPwd: jahia env name to fetch info from
+    #   - jcustomerPwdB64: Jcustomer password (base64 encoded)
     - cmd[proc]: |-
         CONFIG_FILE="/data/digital-factory-data/karaf/etc/org.jahia.modules.jexperience.settings-global.cfg"
         URL=$(grep jexperience.jCustomerURL $CONFIG_FILE | cut -d'=' -f2 | sed 's/ //g' |sed 's/https\?...//g')
```
Comment (laurentfufu) [inline mixins/jahia.yml:794]: what if the character `=` is part of the password ? :trollface:

File: mixins/jahia.yml [hunk 3]
```diff
@@ -802,7 +802,7 @@ actions:
     - cmd[proc]: |-
         CONFIG_FILE="/data/digital-factory-data/karaf/etc/org.jahia.modules.jexperience.settings-global.cfg"
         PWD=$(grep jexperience.jCustomerPassword $CONFIG_FILE | cut -d'=' -f2 | sed 's/ //g')
```
Comment (laurentfufu) [inline mixins/jahia.yml:804]: what if the character `=` is part of the password ? :trollface:

Comment (adoux) [reply mixins/jahia.yml]: Fixed

**Comment by adoux** [comment]
@kevinvandel, @laurentfufu, I've changed every occurence of plain password to base64, I tested:
* Creation of an environment
* Scale out of Unomi
* Link to Jahia
* Change Jcustomer password

Everything works with that password: `P\a$$$w=a=$$o$r:d1'2"3""""-''\\\` so I think we are pretty good now
* (it's worth mentioning that jCustomer doesn't work with that password, but IMO it is a product limitation)

File: README.md [hunk 4]
```diff
@@ -354,17 +354,9 @@ This *upgrade* package aims at upgrading Jahia version by redeploying Jahia node
 
 This manifest will update the _jExperience_ module's configuration with the provided password.
 
-| parameter | comment                                |
-|-----------|----------------------------------------|
-| pwd       | The password _jExperience_ have to use |
-
-#### jahia/set-jahia-root-password.yml
```
Comment (kevinvandel) [inline README.md:361]: Why did you delete this ?

Comment (adoux) [reply README.md:361]: Double with lines 324

**Review by kevinvandel — APPROVED** [review]
